src/lib/src/footer/footer.component.css

```diff
@@ -133,16 +110,7 @@ footer {
  * Social icons
  */
 ul.social {
-  display: -webkit-box;
```

Comment:
I'm guessing this was pre-existing, and will probably get fixed when mobile gets implemented ... but on smaller widths, the social icons get pretty close to the columns above:

![image](https://user-images.githubusercontent.com/1410587/40945399-d4f828e8-6815-11e8-9d2b-3735eb646add.png)

Reply:
I will definitely note this for the responsive work to be done on metrics. I also wondered if the social icons should move to the left and match the same set of corporate icons (same for columns of links.) The footers are all blank in the designs, so I don't think it has been thought out yet.